JS/Lesson14/emrah/script.js

```diff
@@ -7,128 +7,143 @@ const x = 5;
 const y = 3;
 const z = x + y;
 
-console.log('Exercise 1:', z);
+console.log("Exercise 1:", z);
```

Comment:
Hi Emrah, you'll need to remove this file from the pull request. To do that, copy the content of this file from the "main" branch here: https://github.com/frontend-from-0/110523-exercises/blob/main/JS/Lesson14/emrah/script.js
and paste it to the JS/Lesson14/emrah/script.js locally on the emrah-lesson15 branch. This would remove any difference for the Lesson14 file so it will not show up here.